reference/masterfiles-policy-framework/cfe_internal-federation-federation.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: cfe_internal/enterprise/federation/federation.cf
+published: true
+sorting: 80
+tags: [reference, MPF, cfe_internal, federation]
+---
+
+[%CFEngine_library_include(cfe_internal/enterprise/federation/federation)%]
```

Comment:
something like this @nickanderson ?

Reply:
looks right, our macros are a bit weird.

Reply:
yeah looks right